Assets/Plugins/Source/Core/EOSManager_DynamicLoading.cs

```diff
@@ -20,15 +20,10 @@
 * SOFTWARE.
 */
 
-//#define USE_MANUAL_METHOD_LOADING
-
-// TODO: confirm that the overlay doesn't work on macOS
 #if UNITY_STANDALONE_WIN && !UNITY_EDITOR && !UNITY_IOS && !UNITY_STANDALONE_OSX
 #define USE_EOS_GFX_PLUGIN_NATIVE_RENDER
 #endif
 
-//#define USE_WSA_DYNAMIC_LOAD
-
 #if UNITY_64 || UNITY_WSA
```

Comment:
If we are removing Windows Store Application support, we probably want to get rid of the second part of this conditional.